.travis.yml

```diff
@@ -45,8 +45,7 @@ jobs:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.16
```

Comment:
I'm personally OK with dropping 2.16 support. I couldn't find an `adopted-ember-addons` policy for this.

Supported LTS versions at time of writing are: 2.18, 3.4, and 3.8.

Reply:
I'm also not sure. But maybe we just do the Ember CLI upgrade before next release also, which will cause a breaking change anyway due to dropping support for node 6?